### PR TITLE
INSIGHTS-329 - Convert kyverno plugin to go and fix policy title

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What alternative solution should we consider, if any?

[Internal Ticket INSIGHTS-329](https://fairwinds.myjetbrains.com/youtrack/issue/INSIGHTS-329)

### CODE REVIEW

File: plugins/kyverno/cmd/main.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+type Client struct {
+	RestMapper       meta.RESTMapper
+	DynamicInterface dynamic.Interface
+}
+
+func main() {
+	logrus.Info("Starting Kyverno plugin")
+	client, err := getKubeClient()
+	if err != nil {
+		panic(err)
```
Comment (vitorvezani) [inline plugins/kyverno/cmd/main.go:28]: Expected errors should not `panic`, as an alternative, use `logrus.Fatalf("error on doing xyz: %w, err)` instead.

File: plugins/kyverno/cmd/main.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+		p.Object["results"] = violations
+		allViolations = append(allViolations, p.Object)
+	}
+	return allViolations, nil
+}
+
+func createPoliciesTitleAndDescriptionMap(client *Client) (map[string]interface{}, error) {
+	clusterPoliciesMetadata, err := client.ListPolicies(context.Background(), "ClusterPolicy", client.DynamicInterface, client.RestMapper)
+	if err != nil {
+		return map[string]interface{}{}, err
```
Comment (vitorvezani) [inline plugins/kyverno/cmd/main.go:98]:
```suggestion
		return nil, err
```

File: plugins/kyverno/go.mod [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	golang.org/x/oauth2 v0.21.0 // indirect
+	golang.org/x/sys v0.21.0 // indirect
+	golang.org/x/term v0.21.0 // indirect
+	golang.org/x/text v0.16.0 // indirect
+	golang.org/x/time v0.5.0 // indirect
+	google.golang.org/protobuf v1.34.2 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/api v0.31.0 // indirect
+	k8s.io/apimachinery v0.31.0
```
Comment (vitorvezani) [inline plugins/kyverno/go.mod:62]: Kubernetes libs bring so many indirect dependencies 🤕

File: plugins/kyverno/cmd/main.go (hunk 2 above)
Comment (jdesouza) [reply plugins/kyverno/cmd/main.go]: done

File: plugins/kyverno/cmd/main.go (hunk 1 above)
Comment (jdesouza) [reply plugins/kyverno/cmd/main.go]: done

**Review by vitorvezani — APPROVED** [review]

**Pull request merged**